pkg/skyenv/values.go

```diff
@@ -169,3 +169,9 @@ func Version() string {
 	}
 	return v
 }
+
+// HomePath gets the current user's home folder
+func HomePath() string {
```

Comment:
What if this returns an error? Will the visor start? If so, where will it start?

Reply:
You are correct, this could cause an odd configuration to be produced if HomePath returns an empty string.

I've now implemented a check on the config gen command to not offer the `-u --user` flag if HomePath returns an empty string. Similarly to how the visor determines if the -u flag should be an option based on the presence of the config or not.

if HomePath returned empty before these changes it _would have been_ obvious from the config gen help menu as it's displayed in the flag description.